foreman/start-hydra.sh

```diff
@@ -5,6 +5,12 @@ while ! pg_isready -h $(pwd)/.hydra-data/postgres -p 64444; do sleep 1; done
 
 createdb -h $(pwd)/.hydra-data/postgres -p 64444 hydra
 
+# create a db for the default user. Not sure why, but
+# the terminal is otherwise spammed with:
+#
+#     FATAL:  database "USERNAME" does not exist
+createdb -h $(pwd)/.hydra-data/postgres -p 64444 "$(whoami)" || true
```

Comment:
Not sure if these ``$(pwd)`` shoud also be converted.

Reply:
Nah, I have a PR brewing that'll clean these up. Thanks, though.